tests/nfdiv/nightly/verifyFinalOrderJourney.test.js

```diff
@@ -20,92 +21,26 @@ Feature('NFD - Create a single Case and move it to Final Order Pronounced');
-
-  const shareACase = await updateRoleForCase(user.RS,caseNumber,'APPTWOSOLICITOR');
-
-  const caseSharedToRespSolicitor = await shareCaseToRespondentSolicitor(user.RSA,caseNumber);
-  assert.strictEqual(JSON.parse(caseSharedToRespSolicitor).status_message, 'Roles [APPTWOSOLICITOR] from the organisation policies successfully assigned to the assignee.');
-
-  console.log('~~~~~~~~~ Case with Id ' + caseNumber +' has been SUCCESSFULLY SHARED  to Respondent Solicitior');
-
-  const draftAoS = await updateNFDCaseInCcd(user.RS,caseNumber, events.DRAFT_AOS,'data/ccd-draft-aos.json');
-  verifyState(draftAoS, states.AOS_DRAFTED);
-
-  const submitAoS = await updateNFDCaseInCcd(user.RS,caseNumber, events.SUBMIT_AOS,'data/ccd-submit-aos.json');
-  verifyState(submitAoS, states.HOLDING);
-
-  // To Move case from 20WeekHolding to AwaitingConditionalOrder  .... Call CCD API to mimic the cron job.
-  // and set the dueDate to null ..(See SystemProgressHeldCasesTask in nfdiv-case-api)
-
-  const awaitingConditionalOrder = await moveFromHoldingToAwaitingCO('data/await-co-data.json',caseNumber);
-  assert.strictEqual(JSON.parse(awaitingConditionalOrder).state, 'AwaitingConditionalOrder');
-
-  // Draft CO
-  const draftConditionalOrder = await updateNFDCaseInCcd(user.SOLS,caseNumber, events.SOLS_DRAFT_CO,'data/ccd-draft-co.json');
-  verifyState(draftConditionalOrder, stateDisplayName.CONDITIONAL_ORDER_DRAFTED);
-
-  // Submit CO
-  const awaitingLegalAdvisorReferral = await updateNFDCaseInCcd(user.SOLS,caseNumber, events.SUBMIT_CO,'data/ccd-submit-co.json');
-  verifyState(awaitingLegalAdvisorReferral, states.AWAITING_LEGAL_ADVISOR_REFERRAL);
-
-  // Moves case to Listed;AwaitingPronouncement state
-  const listedAwaitingPronouncement = await updateNFDCaseInCcd(user.LAD,caseNumber, events.LEGAL_ADVISOR_MAKE_DECISION,'data/ccd-la-make-decision.json');
-  verifyState(listedAwaitingPronouncement, states.AWAITING_PRONOUNCEMENT);
-
-  //Note:Important: BulkCase with just ONE CaseParty reference . Purely for e2e purpose Only and to enable testing of the Pages that follow it.
-  const bulkCaseReferenceId = await moveCaseToBulk('data/bulk-case-data.json',caseNumber);
-  // verifyState(bulkCaseReferenceId, states.BULK_CASE_LISTED_CREATED);
-
-  // kasi
-  const scheduleBulkList = await bulkCaseListSchedule(user.CA, bulkCaseReferenceId,caseNumber, 'caseworker-schedule-case', 'data/bulk-case-list-schedule-data.json');
-  verifyState(scheduleBulkList, 'Listed');
-
-  await I.wait(60);
-
-  const pronounceBulkList = await bulkCaseListPronounced(user.CA, bulkCaseReferenceId,caseNumber, 'caseworker-pronounce-list', 'data/bulk-case-list-pronounce-data.json');
-  await I.wait(30);
-
-  // poll the Case within tthe bulkPack to check if it in 'Pronounced' state
-  verifyState(pronounceBulkList, 'Pronounced');
-
-  const moveCaseToPronounced = await moveCaseToConditionalOderPronounced('system-pronounce-case','data/conditional-order-pronounced.json',caseNumber);
```

Comment:
moveCaseToConditionalOderPronounced  - typo here , missing a  letter 'r' .... should be moveCaseToConditionalOrderPronounced